modules/core/src/main/scala/ch/epfl/bluebrain/nexus/admin/core/resources/Resources.scala

```diff
@@ -120,31 +122,26 @@ class Resources[F[_], A: IdResolvable: TypeFilterExpr](agg: Agg[F], sparqlClient
     *
     * @param id     the identifier of the resource
     * @param rev    the last known revision of the resource instance
-    * @param tags   the tags added to the consequent [[ResourceEvent]] which might be created as a result of this operation
-    * @param persId the unique persistence ID of the resource
     * @return a [[RefVersioned]] instance wrapped in the abstract ''F[_]'' type
     *         if successful, or a [[ch.epfl.bluebrain.nexus.admin.core.Fault]] wrapped within ''F[_]'' otherwise
     */
-  def deprecate(id: A, rev: Long)(tags: Set[String], persId: String)(
-      implicit ctx: CallerCtx,
-      @silent perms: HasWriteProjects): F[RefVersioned[A]] =
-    evaluate(DeprecateResource(id, rev, ctx.meta, tags + persId), persId, s"Deprecate res '$id'")
+  def deprecate(id: A, rev: Long)(implicit ctx: CallerCtx): F[RefVersioned[A]] =
```

Comment:
you need an evidence that you can call that method, otherwise anybody can.
`@silent perms: HasWriteProjects`